packs/data/feats.db/stone-face.json

```
@@ -19,11 +19,35 @@
         "level": {
             "value": 1
         },
-        "location": "",
+        "location": null,
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "hideIfDisabled": "true",
+                "key": "FlatModifier",
+                "predicate": {
+                    "any": [
+                        "action:demoralize"
+                    ]
```

```suggestion
                    "any": [
                        "action:demoralize",
                        "action:coerce"
                    ]
```

Might as well tag both intimidation skill actions.